packages/darklang/stdlib/string.dark

```diff
@@ -40,11 +40,61 @@ module Darklang =
       let isEmpty (s: String) : Bool = s == ""
 
 
+      /// Iterate over each Character (EGC, not byte) in the string, performing the
+      /// operation in <param fn> on each one.
+      let map (s: String) (fn: Char -> Char) : String = Builtin.String.map s fn
+
+
+      /// Returns the list of Characters (EGC, not byte) in the string
+      let toList (s: String) : List<Char> = Builtin.String.toList s
+
+
+      /// Replace all instances on <param searchFor> in <param s> with <param replaceWith>
+      let replaceAll (s: String) (searchFor: String) (replaceWith: String) : String =
+        Builtin.String.replaceAll s searchFor replaceWith
+
+
+      /// Returns the string, uppercased
+      let toUppercase (s: String) : String = Builtin.String.toUppercase s
+
+
+      /// Returns the string, lowercased
+      let toLowercase (s: String) : String = Builtin.String.toLowercase s
+
+
+      /// Returns the length of the string
+      let length (s: String) : Int = Builtin.String.length s
+
+
+      /// Concatenates the two strings by appending <param s2> to <param s1> and returns the joined string.
+      let append (s1: String) (s2: String) : String = $"{s1}{s2}"
+
+
       /// Concatenates the two strings by prepending <param s2> to <param s1> and
       /// returns the joined string
       let prepend (s1: String) (s2: String) : String = s2 ++ s1
 
 
+      /// Turns a string into a prettified slug, including only lowercased
+      /// alphanumeric characters, joined by hyphens
+      let slugify (s: String) : String = Builtin.String.slugify s
```

Comment:
I'd rather this be deleted than builtin

Reply:
Can it be implemented without regex? If not, maybe we comment it out until we have regex?